datasource/src/main/scala/org/locationtech/rasterframes/datasource/tensor/TensorDataSource.scala

```diff
@@ -107,6 +114,11 @@ object TensorDataSource {
         reader.option(TensorDataSource.BAND_INDEXES_PARAM, bandIndexes.mkString(","))
       )
 
+    def usePatternExpansion(flag: Boolean = true): TaggedReader =
```

Comment:
What happens if this default is set to true while the above default for `PatternToRasterSources` is set to false? (I assume if left unspecified it will be false?)